recipe/meta.yaml

```
@@ -16,7 +16,7 @@ source:
 
 build:
   number: 1
-  skip: true  # [aarch64 or osx or win]
+  skip: true  # [aarch64 or osx or win or (linux and s390x)]
```

```suggestion
  skip: true  # [osx or win or (linux and (aarch64 or s390x))]
```
Logically, it would look better

---

it's put like this to make the diff from conda-forge easy to understand. Although I get where you're coming from.